services/coincap/coincap-base.js

```diff
@@ -0,0 +1,22 @@
+import { BaseJsonService } from '../index.js'
+
+export default class BaseCoincapService extends BaseJsonService {
+  static category = 'other'
+
+  static defaultBadgeData = { label: 'Bitcoin' }
```

Comment:
Sorry I should have spotted this the first time round. There's a couple of things here: Labels should always be lowercase, so we should lowercase `asset.name` when we set it. Also, the default label should really be something generic. i.e: if you pass 'ethereum' as your param, get an error and the error is `bitcoin | something went wrong`, that would be odd. I think `coincap` should be the default label as that is the service name.